server/service/team_policies.go

```diff
@@ -193,10 +202,31 @@ func (svc Service) DeleteTeamPolicies(ctx context.Context, teamID uint, ids []ui
 	if len(ids) == 0 {
 		return nil, nil
 	}
-	ids, err := svc.ds.DeleteTeamPolicies(ctx, teamID, ids)
+	// Is there a better approach to gather policy names so we can include them in the activity records?
```

Comment:
Similar as before. Even if you use this method, I would abstract it away into a function and use it in both places.